README.md

```diff
@@ -1,57 +1,7 @@
-# Northcoders News API
+When running the development database use:
 
-## Background
+PGDATABASE=nc_news
 
-We will be building an API for the purpose of accessing application data programmatically. The intention here is to mimic the building of a real world backend service (such as reddit) which should provide this information to the front end architecture.
+When running the test database use:
```

Comment:
This needs some explanation of how to set up the .env files, i.e. name them appropriately .env.development etc.